playground/backend/internal/cloud_bucket/precompiled_objects.go

```diff
@@ -44,11 +44,12 @@ const (
 )
 
 type ObjectInfo struct {
-	Name        string
-	CloudPath   string
-	Description string                   `protobuf:"bytes,3,opt,name=description,proto3" json:"description,omitempty"`
-	Type        pb.PrecompiledObjectType `protobuf:"varint,4,opt,name=type,proto3,enum=api.v1.PrecompiledObjectType" json:"type,omitempty"`
-	Categories  []string                 `json:"categories,omitempty"`
+	Name            string
+	CloudPath       string
+	Description     string                   `protobuf:"bytes,3,opt,name=description,proto3" json:"description,omitempty"`
+	Type            pb.PrecompiledObjectType `protobuf:"varint,4,opt,name=type,proto3,enum=api.v1.PrecompiledObjectType" json:"type,omitempty"`
+	Categories      []string                 `json:"categories,omitempty"`
+	PipelineOptions string                   `protobuf:"bytes,3,opt,name=pipeline_options,proto3" json:"pipeline_options,omitempty"`
```

Comment:
Now it the pipeline_options are empty, they will be omitted. Is it fine for PrecompiledObject message?

Reply:
PrecompileObject gets all these values from `meta.info` file. So if this file contains or doesn't contain this field it would be ok.